### PR TITLE
Added week 4 blog post

### PR DESCRIPTION
**Pull request by m-agour**
Added week 4 blog post which contains some info and a single video.

image

### CODE REVIEW

File: docs/source/posts/2022/2022-07-11-week-4-mohamed.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+What is coming up next week?
+----------------------------
+
+This week I will do the following:
+
+- Start Implementing slerp interpolation using quaternions in both Python and GLSL. And use slerp to apply camera rotation.
+- Add tests for the previous work.
+- Make a PR to merge the non-shader-based version of the ``Timeline``.
+- Test how shader attributes can be used to hold keyframes for each vertex and benchmark it.
+- Study 'colormaps' and implement some 'colormaps' in GLSL.
```
Comment (ganimtron-10) [inline docs/source/posts/2022/2022-07-11-week-4-mohamed.rst:52]: Can you just recheck whether you want `colormaps` (with ` ) or 'colormaps' (with ' )?

**Review by ganimtron-10 — COMMENTED** [review]
Hi @m-agour ,
Perfect Blog! I found no errors but had a comment which I have mentioned below.
PTAL.

Thanks!

**Review by xtanion — COMMENTED** [review]
Nice blog @m-agour , Everything looks good to me, there are no typos or grammatical errors in the blog. Good job!

File: docs/source/posts/2022/2022-07-11-week-4-mohamed.rst (hunk 1 above)
Comment (m-agour) [reply docs/source/posts/2022/2022-07-11-week-4-mohamed.rst:52]: It's not a code so I might just make it _italic_

**Pull request merged**